node-deb

```diff
@@ -175,43 +175,49 @@ while [ -n "$1" ]; do
     --template-control)
       # HELPDOC: Override Debian control template (default: 'node_deb.templates.control' from package.json then built-in)
       zero_check "$value" "$param"
-      template_control="$param"
+      template_control="$value"
```

Comment:
I can't believe these have all been broken for so long and no one noticed. Good eye.